Library/Homebrew/os/mac/mach.rb

```diff
@@ -90,7 +90,7 @@ def dynamically_linked_libraries(except: :none, resolve_variable_references: tru
   end
 
   def rpaths(resolve_variable_references: true)
-    names = macho.rpaths.uniq
+    names = macho.rpaths
```

Comment:
CC @gerlero in case calling `#uniq` here is important for the other changes from #15571.

Reply:
I remember putting that `#uniq` call there; my thinking was that the new method (`#rpaths`) should probably follow what `#dynamically_linked_libraries` does:
 https://github.com/Homebrew/brew/blob/bd5bd8437f96bce1576fbc46d64203f41fd848c7/Library/Homebrew/os/mac/mach.rb#L82-L89

It isn't needed for anything else and can be dropped.

Reply:
Great; that's what I thought but wanted to make sure. Thanks!